src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -184,5 +213,17 @@ static void checkArg(string arg, string name)
             return (failed ? -1 : 0);
 
         }
+        
+        private static ProcessHelper SetUpIteration(string command, string args, string workingDir)
+        {
+            var procHelper = new ProcessHelper()
```

Comment:
Need more feedback on the properties of the iteration set up process; not sure whether to make Timeout and MeasurementDelay settable.

Reply:
If I understand correctly, MeasurementDelay shouldn't be needed here because there is no measurement happening during setup.  Timeout could be interesting because you want to make sure that setup doesn't run forever.  Is there a default timeout?

Reply:
Yes the default timeout is 60 seconds. I assume the setup won't be an GUIApp and the workingDir is the same as the main iteration.